source/codegen/metadata/nirfmxinstr/enums_addon.py

```diff
@@ -4,3 +4,6 @@
 enums_override_metadata = {
 }
 
+enums_validation_suppressions = {
+    "FrequencyReferenceExportedTerminal": ["ENUMS_SHOULD_NOT_HAVE_DUPLICATE_VALUES"]
```

Comment:
What is this bit about? I might be missing something but I don't see this 'enums_validation_suppressions' string referenced anywhere except the `__init__.py` and `enums_addon.py` for a few drivers.

Reply:
Yeah! This was tricky to track down.

1. There's a rule in metadata_validation to require no duplicate enum values.
2. There's a way to override that rule.
3. BUT how does it work!?

I'm guessing that the bit you missed was the same one that took me forever to find: [rule_is_suppressed](https://github.com/ni/grpc-device/blob/d6fddbe4f42eb9f4c36fd3d037385cf7dc1a6088/source/codegen/metadata_validation.py#L155).
 So basically the "enums" prefix means it applies to the `enums.py` metadata file